### DISCUSSION TITLE
useSeoMeta and useServerSeoMeta, differences on Seo?

### DISCUSSION THREAD

**Discussion by notflip** [nuxt/nuxt#25357] [Questions]
Do both of these work exactly the same way? How about for search engines and crawlers? 

I’m in a situation where I can’t use useServerSeoMeta and I’m wondering what the downside will be compared to using the server version rendering my meta tags? Thanks!

**Comment by TheAlexLichter** [comment]
Both "do the same", but `useServerSeoMeta` *only* updates on the server-side (so not when navigating on the client-side).

**Reply by notflip** [reply]
Thanks, @manniL that I know, I'm wondering what the effect is, or the upside/downside of using one vs the other

**Reply by TheAlexLichter** [reply]
The "upside": If you can exclusively use `useServerSeoMeta` you save a few kB because your client bundle does not contain `useSeoMeta` anymore (+ doesn't have to hydrate the meta tags). That's a minor perf improvement. 👍
